plugins/inputs/win_services/win_services.go

```diff
@@ -108,7 +110,7 @@ type ServiceInfo struct {
 
 func (m *WinServices) Init() error {
 	var err error
-	m.servicesFilter, err = filter.NewIncludeExcludeFilter(m.ServiceNames, nil)
+	m.servicesFilter, err = filter.NewIncludeExcludeFilter(m.ServiceNames, m.IgnoredNames)
```

Comment:
To create a unit test for this, you could create a new function similar to 'TestGatherContainsTag' but then test to see if it doesn't contain the tag that you are trying to ignore, the `testutil.Accumulator` comes with a function called `AssertDoesNotContainsTaggedFields` you can use to do this. Hopefully that helps get you started!

Reply:
Test added.

Reply:
Great Job!